.github/workflows/golangci-lint.yml

```diff
@@ -6,13 +6,13 @@ jobs:
   build:
     strategy:
       matrix:
-        go-version: [1.16.x, 1.17.x, 1.18.x]
+        go-version: [1.18.x, 1.19.x, 1.20.x]
```

Comment:
Nit: probably we can drop 1.18.x here as it's no longer supported.

Reply:
Yeah, good point